components/calendar/src/arithmetic.rs

```diff
@@ -10,10 +10,7 @@ pub mod week_of {
 
     /// Information about how a given calendar assigns weeks to a year or month.
     #[derive(Clone, Copy, Debug)]
-    #[cfg_attr(
-        feature = "provider_serde",
-        derive(serde::Serialize, serde::Deserialize)
-    )]
+    #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Suggestion (optional): Inside the code, in most cases, you can already use `feature = "serde"`. It's only in the Cargo.toml files that we're being restricted. I guess this doesn't work if one of the fields needs a feature enabled on its crate.

Reply:
Yeah it's better to be consistent here. Furthermore, it'll be moot once we start using `dep:serde`